frontend/src/App.js

```diff
@@ -13,11 +16,21 @@ function App() {
         },
         [])
 
+    const onDraftUpdated = (updatedDraft) => {
+        setDraft(updatedDraft);
+    }
 
     return (
         <div>
```

Comment:
Whats the use of this unstyled container? It looks like the could be elemenated. U could use fragments instead.